components/financial_assistance/app/views/financial_assistance/shared/_fa_left_nav.html.erb

```diff
@@ -67,9 +70,9 @@
           <li class="list-box step-tabs <%= li_nav_classes_for(:health_coverage) %> applicant-nav">
             <% if @applicant.embedded_document_section_entry_complete?(:health_coverage)%>
               <span class="fa fa-check applicant-check"><span class="hide"><%=l10n("completed")%></span></span>
-              <%= link_to_with_noopener_noreferrer('Health Coverage', financial_assistance.application_applicant_benefits_path(@application, @applicant), class: "") %>
```

Comment:
the scanner didn't throw any XSS warnings here, but assuming the two flagged uses of `link_to` in the diff were true positives, this one should probably be account for too as it is a custom helper built on top of `link_to`